packages/report-server/src/__tests__/reportBuilder/transform/parser/functions.test.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { TransformParser } from '../../../../reportBuilder/transform/parser';
+
+describe('functions', () => {
+  describe('basic', () => {
+    describe('value', () => {
+      it('returns a string when passed to it', () =>
+        expect(new TransformParser().evaluate("=value('some string')")).toBe('some string'));
```

Comment:
All these tests are basically the same as the existing `functions.test.ts` with a few added. However evoking them via the `TransformParser` is a much better test as it validates how these will evaluate when used in reports